src/common/constants.ts

```diff
@@ -1,3 +1,10 @@
 export const SIDE_DRAWER_WIDTH = 200;
 export const DATE_FORMAT = 'DD/MM/YY';
 export const TIME_FORMAT = 'HH:mm';
+export const REACT_APP_GLIFIC_REGISTRATION_API =
+  process.env.REACT_APP_GLIFIC_API + '/v1/registration';
+export const REACT_APP_GLIFIC_AUTHENTICATION_API =
+  process.env.REACT_APP_GLIFIC_API + '/v1/registration/send_otp';
+export const REACT_APP_GLIFIC_CHAT_REDIRECT = 'http://localhost:3000/chat';
```

Comment:
We should not hardcode this here. I don't think we need redirect URL. We can directly send the user to the route.